io/net/htx_nic_devices.py.data/README.txt

```diff
@@ -5,10 +5,14 @@ ready for the execution and also updates bpt file with necessary
 data. Once configuration is done test starts executing the stress
 test on all the interfaces provided in input yaml file
 
-User can mention the time limit in hours to execute.
+User can mention the time limit to be calculated in one tenth of an
+hour. For eg, value 1 is 60 minutes
```

Comment:
Yes it was. That could have been not changed and left as is but just thought I'll add a more detailed description to what was existing.